c2corg_ui/static/js/maincontroller.js

```diff
@@ -92,3 +97,9 @@ app.MainController.prototype.isPath = function(path) {
 app.MainController.prototype.animateHeaderIcon = function(e) {
   app.utils.animateHeaderIcon(e);
 };
+/**
+ * @export
+ */
+app.MainController.prototype.updateMapResize = function() {
+  this.scope_ .$root.$emit('updateMapResize');
```

Comment:
This is WIP so I let myself some consistencies :p 

See my comment above: I could not access the `mapCtrl` in those buttons probably because they are outside the directive whereas the `mainCtrl` can be accessed anywhere. This was one of my concerns and i'm not sure how to deal with it correctly.